app/routes/users/components/UserProfile.js

```diff
@@ -92,38 +93,47 @@ const BadgeTooltip = ({
   return `${group.name} (${startYear} - ${endYear})`;
 };
 
-const GroupBadge = ({
-  group,
-  start,
-  end
-}: {
-  group: Group,
-  start: Dateish,
-  end?: ?Dateish
-}) => {
+const GroupBadge = ({ memberships }: { memberships: Array<Object> }) => {
+  const activeMemberships = memberships.find(membership => membership.isActive);
+  const [{ abakusGroup }] = memberships;
+  // $FlowFixMe
+  const sorted = orderBy(memberships, membership =>
+    moment(membership.startDate || membership.createdAt)
+  );
+  const [firstMembership] = sorted;
+  const [lastMembership] = sorted.reverse();
```

Comment:
You don't need to reverse the whole list to get the last element:
```
const firstMembership = sorted[0];
const lastMembership = sorted[sorted.length - 1];
```
I would also rename `sorted` to something like `sortedMemberships`.